applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -1016,21 +1022,28 @@ Syntax
 ~~~~~~
 
 ::
+   .. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is enabled.
+   #XTCPSVR=<op>[,<port>,<timeout>,[sec_tag]]
 
-   #XTCPSVR=<op>[<port>[,<sec_tag>]]
+   .. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is disabled.
+   #XTCPSVR=<op>[,<port>[,<sec_tag>]]
```

Comment:
Redundant line?
.. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is disabled.

Reply:
**_.. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is disabled._** should be removed.
It might be something wrong when I port ipv6.